package/MDAnalysis/analysis/rms.py

```diff
@@ -235,13 +235,16 @@ def rmsd(a, b, weights=None, center=False, superposition=False):
 
     Example
     -------
-    >>> u = Universe(PSF,DCD)
+    >>> import MDAnalysis as mda
+    >>> from MDAnalysis.analysis.rms import rmsd
+    >>> from MDAnalysis.tests.datafiles import PSF, DCD
+    >>> u = mda.Universe(PSF, DCD)
     >>> bb = u.select_atoms('backbone')
     >>> A = bb.positions.copy()  # coordinates of first frame
-    >>> u.trajectory[-1]         # forward to last frame
+    >>> _ = u.trajectory[-1]  # forward to last frame
     >>> B = bb.positions.copy()  # coordinates of last frame
     >>> rmsd(A, B, center=True)
-    3.9482355416565049
+    6.838544558398293
 
     .. versionchanged: 0.8.1
```

Comment:
```suggestion

    .. versionchanged: 0.8.1
```
This is a slightly separate issue, but it looks like the versionchanged entries are being ignored because doctest need a blank line and so do versionchanged entries. I'll self merge this fix and see if it fixes the problem.

Reply:
That didn't fix it, I'm going to go ahead and shift the versionchanged problem to a separate issue, it's an issue that predates this fix anyways.